src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/CyclomaticComplexityCheck.java

```diff
@@ -60,15 +60,18 @@
  * <ul>
  * <li>
  * Property {@code max} - Specify the maximum threshold allowed.
+ * Type is {@code int}.
  * Default value is {@code 10}.
  * </li>
  * <li>
  * Property {@code switchBlockAsSingleDecisionPoint} - Control whether to treat
  * the whole switch block as a single decision point.
+ * Type is {@code boolean}.
  * Default value is {@code false}.
  * </li>
  * <li>
  * Property {@code tokens} - tokens to check
+ * Type is {@code subsetOfTokenTypes}.
```

Comment:
javadoc is  class descrition, we have to be exact on this level.
It might be weird, but we need real type in setter so - `int[]`.

metadata generator and xdoc generator will make it subsetOfTokenTypes as it is very special property. The same will be for all other special types.

Reply:
Done